packages/core/src/common/contribution-filter/contribution-filter.ts

```diff
@@ -50,6 +50,9 @@ export const FilterContribution = Symbol('FilterContribution');
 export interface FilterContribution {
     /**
      * Use the registry to register your contribution filters.
+     * Note that filtering contributions based on their class (constructor) name is discouraged.
+     * Class names are minified in production builds and therefore not reliable.
+     * Use instance of checks or direct constructor comparison instead.
```

Comment:
It might make sense to add an example code snippet here:
```suggestion
     * Note that filtering contributions based on their class (constructor) name is discouraged.
     * Class names are minified in production builds and therefore not reliable.
     * Use instance of checks or direct constructor comparison instead:
     * 
     * ```ts
     * registry.addFilters('*', [
     *     contrib => !(contrib instanceof SampleFilteredCommandContribution)
     * ]);
     * ```
```